shared/network/2.x.md

```diff
@@ -499,6 +499,8 @@ Note that:
 
 Additionally, a `/mnt/boot/system-proxy/no_proxy` file can be added with a newline-separated list of IPs or subnets to not route through the proxy.
 
+We've published an example [here](https://github.com/balenalabs/proxy-tunnel) demonstrating how to implement this on devices.
```

Comment:
@ab77 Using "here" to point to links is usually not the best practice for documentation. You can use this instead. 

`Check this [example](https://github.com/balenalabs/proxy-tunnel) of implementing proxy-tunnel on devices.`